module-1/README.md

```diff
@@ -63,55 +117,255 @@ export const environment = {
-We'll use AWS Amplify later in the tutorial to add authentication, user sign-in, and API access, but the AWS Amplify CLI also includes a feature for `hosting`.
+For the purposes of this workshop we will use TypeScript as our language.  So execute the following command
+
+```bash
+cdk init app --language=typescript
+```
 
-**It's important to note:** Only choose one method of deployment – either use the deployment scripts or AWS Amplify.
+This command will initialise a new CDK application.
 
-Follow these minimum steps for working with the AWS Amplify CLI, or view the full [Getting Started guide](https://aws-amplify.github.io/amplify-js/media/quick_start?platform=purejs).
+### Implementation
+
+Now, let's implement the code to host our web application.  Open the `CDK` folder in VS Code.
+
+The standard structure of an AWS CDK application consists of a `bin` folder and a `lib` folder.
+
+* The `bin` folder is where we will define the entry point for the CDK application.
+* The `lib` folder is where we will define all our workshop infrastructure components.
+
+before we start implementing our code, let's get typescript to watch for file changes and compile the javascript whenever a change is detected by running the following command from the terminal.
+
+```bash
+npm run watch
+```
 
-* Install the AWS Amplify CLI:
-    * npm install -g @aws-amplify/cli
-    * amplify configure
-* Navigate to the `frontend` directory.
-* Run `amplify init` in the `frontend` directory and select the following options:
-    * Choose your default editor: `Visual Studio Code`
-    * Choose the type of app that you're building: `javascript`
-    * What javascript framework are you using: `angular`
-    * Source Directory Path: `src`
-    * Distribution Directory Path: `dist`
-    * Build Command: `npm run build -- --prod`
-    * Start Command: `npm start`
-* Run `amplify add hosting`
-    * Select `PROD (S3 with CloudFront using HTTPS)`
-    * Provide a [globally unique name](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html) for your hosting bucket
-* Run `amplify publish`
+#### Module 1 Files
+
+Next, Within the lib folder you should find a file with the default name of `cdk-stack.ts`.  Rename this file to `Module1Stack.ts`.
+
+Open the file `Module1Stack.ts` in VS Code and ename the class to `Module1Stack`, as illustrated below.
+
+```typescript
+import cdk = require('@aws-cdk/cdk');
+
+export class Module1Stack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // The code that defines your stack goes here
+  }
+}
+```
+
+Following the change of filename and classname, you should now update the references in the `cdk.ts` file, as such.
+
+```typescript
+#!/usr/bin/env node
+import 'source-map-support/register';
+import cdk = require('@aws-cdk/cdk');
+import { Module1Stack } from '../lib/Module1Stack';
+
+const app = new cdk.App();
+new Module1Stack(app, 'Module1');
```

Comment:
Something to consider: If the directory name would have been `module1` instead of `cdk`, the stack name would have been `Module1Stack` automatically.

Reply:
Agreed, but in this case we will have one cdk folder for all components we create across the modules, rather than folders for module1,2,3 etc.